lib/rust/ensogl/component/text/src/typeface/glsl/glyph_mac.glsl

```diff
@@ -1,3 +1,6 @@
+// A factor describing much the bold letters will be fattened, expressed as the fraction of font size.
+const float BOLD_FATTING = 0.04;
+
```

Comment:
the file for non-mac has more changes, is it intentional?

Reply:
No, not all realignments/refactorings were moved. However, logic changes should be the same.